src/Platform/Microsoft.Testing.Platform/DotnetTest/IPC/Constants.cs

```diff
@@ -3,6 +3,11 @@
 
 namespace Microsoft.Testing.Platform.IPC;
 
+internal static class Constants
+{
+    internal const string DotnetTestCliProtocol = "dotnettestcli";
```

Comment:
I'd like to have this one inside the platform command line so we have everything in the same place when we check what's we expose.